<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>XXX公司</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="title">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(218, 218, 218);</string>
        </property>
        <property name="text">
         <string>员工信息表</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="gridStyle">
         <enum>Qt::DashDotLine</enum>
        </property>
        <property name="cornerButtonEnabled">
         <bool>false</bool>
        </property>
        <property name="rowCount">
         <number>5</number>
        </property>
        <property name="columnCount">
         <number>5</number>
        </property>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="form_page_bar" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_21">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="form_first_page">
           <property name="text">
            <string>首页</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="form_previous_page">
           <property name="text">
            <string>上一页</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="form_page_num">
           <property name="text">
            <string>[50/200]页</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="form_next_page">
           <property name="text">
            <string>下一页</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="form_last_page">
           <property name="text">
            <string>尾页</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
